ppx/mel_ast_invariant.ml

```diff
@@ -30,34 +30,40 @@ module Warnings = struct
     | Fragile_external of string
     | Redundant_mel_string
     | Deprecated_non_namespaced_attribute
+    | Uncurried_arity0
 
   let kind = function
     | Unused_attribute _ -> "unused"
     | Fragile_external _ -> "fragile"
     | Redundant_mel_string -> "redundant"
     | Deprecated_non_namespaced_attribute -> "deprecated"
+    | Uncurried_arity0 -> "uncurried"
 
   let pp fmt t =
     match t with
     | Unused_attribute s ->
         Format.fprintf fmt
           "Unused attribute [%@%s]@\n\
            This means such annotation is not annotated properly.@\n\
-           For example, some annotations are only meaningful in externals\n"
```

Comment:
Why removing the newline here? I found it was useful to have it (see changes in tests).

Edit: I see other messages don't include it. Maybe the newline could be added separately to have some space between the alert and the actual output when calling `melc`?